pkg/kubelet/apis/config/scheme/testdata/KubeletConfiguration/after/__internal.yaml

```diff
@@ -0,0 +1,93 @@
+Address: ""
```

Comment:
don't test internal API types by serializing them, that is contrary to their intended purpose

Reply:
same goes for all the __internal.yaml files... they should not exist

Reply:
That's a good point that I missed the first time around. We should also clean up the same in kube-proxy from the prior PR.

Reply:
I created https://github.com/kubernetes/kubernetes/pull/86144 to address this issue! PTAL :)